components/fxa-client/src/http_client.rs

```diff
@@ -116,6 +128,28 @@ impl FxAClient for Client {
         self.make_oauth_token_request(config, body)
     }
 
+    fn oauth_tokens_from_session_token(
```

Comment:
We had `oauth_token_from_session_token` and `oauth_token_with_session_token` and no real guidance about which one to use when. I've tried to clarify a bit by distinguishing between the ones that generate a refresh token ("from") and the ones that generate only access tokens ("with").

Reply:
Should we instead prefix these methods with `access_token` or `refresh_token` (the latter implies the former) to make it clearer that e.g. `oauth_tokens_from_session_token` returns both? I'm ok with the current name at the moment.

Reply:
Yes, good idea; I just didn't want to get too carried away in a single PR.